plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1201,6 +1201,65 @@ SecRule REQUEST_FILENAME "@rx /apps/recognize/admin/settings/(?:tensorflow\.(?:c
     ctl:ruleRemoveTargetById=932160;ARGS:json.value,\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# When configuration oauth 2 clients under Administration --> Security --> OAuth 2.0 clients
+# DELETE - Deleting an OAuth client
+SecRule REQUEST_FILENAME "@rx /apps/oauth2/clients(?:/[0-9]+)?$" \
+    "id:9508628,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetById=931130;ARGS:json.redirectUri,\
+    ctl:ruleRemoveTargetById=932236;ARGS:json.name,\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# Disabling automatic deletion of a tag type under Administration --> Flow --> File retention & automatic deletion
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/files_retention/api/v[0-9]/retentions/[0-9]+$" \
+    "id:9508629,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# When enabling/disabling Microsoft/Gmail mail integration via oauth 2 under Settings --> Administration 
+# --> Groupware --> Mail app --> Microsoft integration
+SecRule REQUEST_FILENAME "@rx /apps/mail/api/integration/(?:google|microsoft)$" \
+    "id:9508630,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# Viewing Nextcloud logs
+# Settings --> Administration --> Log Reader
+SecRule REQUEST_FILENAME "@endsWith /apps/logreader/poll" \
+    "id:9508631,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    chain"
+    SecRule ARGS:lastReqId "@rx ^[a-zA-Z0-9]+$" \
```

Comment:
```suggestion
    SecRule ARGS:lastReqId "@rx (?i)^[a-z0-9]+$" \
```

Reply:
don't they both do the same thing, what's the reasoning for the change?

Reply:
Yes, they're equivalent. The latter form is just more in line with the other rules in CRS.